pywinauto/base_wrapper.py

```diff
@@ -336,31 +336,33 @@ def texts(self):
         return texts_list
 
     #-----------------------------------------------------------
-    def children(self, class_name = None, title = None, control_type = None):
+    def children(self, class_name=None, title=None, control_type=None, is_content_element=None):
```

Comment:
I would name it `content_only` to be consistent with `top_level_only` in `find_windows`.